### PR TITLE
Tweaks to bower entry - specifically committing deps

### PR DESCRIPTION
**Pull request by weaverryan**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets | n/a |

This follows after #5159. The biggest addition is a short conversation about committing Bower assets.

Thanks!

### CODE REVIEW

File: cookbook/frontend/bower.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ then run:
 
     $ npm install -g bower
 
-After this command succeeded, run ``bower`` in your terminal to find out if
+After this command finishes, run ``bower`` in your terminal to find out if
```
Comment (xabbuh) [inline cookbook/frontend/bower.rst]: "has finished"?

File: cookbook/frontend/bower.rst [hunk 2]
```diff
@@ -43,6 +43,12 @@ create a ``.bowerrc`` file with a new destination (like ``web/assets/vendor``):
         "directory": "web/assets/vendor/"
     }
 
+.. tip::
+
+    If you're using a frontend build system like `Gulp`_ or `Grunt`_, then
```
Comment (xabbuh) [inline cookbook/frontend/bower.rst]: front-end

File: cookbook/frontend/bower.rst [hunk 3]
```diff
@@ -110,8 +116,31 @@ template like normal CSS/JS:
...
+Should I Git Ignore or Commit Bower Assets?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Currently, you should probably *commit* the assets downloaded by Bower instead
+of adding the directory (e.g. ``web/assets/vendor``) to your ``.gitignore``
+file:
+
+.. code-block:: bash
+
+    git add web/assets/vendor
```
Comment (xabbuh) [inline cookbook/frontend/bower.rst]: missing `$`

File: cookbook/frontend/bower.rst [hunk 4]
```diff
@@ -110,8 +116,31 @@ template like normal CSS/JS:
...
+
+Why? Unlike Composer, Bower currently does not have a "lock" feature, which
+means that there's no guarantee that running ``bower install`` on a different
+server will give you the *exact* assets that you have on other machines.
+For more details, read the article `Checking in fronte-end dependencies`_.
```
Comment (xabbuh) [inline cookbook/frontend/bower.rst]: front-end

File: cookbook/frontend/bower.rst [hunk 5]
```diff
@@ -110,8 +116,31 @@ template like normal CSS/JS:
...
+
+But, it's very possible that Bower will add a lock feature in the future
+(e.g. `bower/bower#1748`_)
```
Comment (WouterJ) [inline cookbook/frontend/bower.rst]: missing full stop: `[...] (e.g. `bower/bower#1748`_).`

File: cookbook/frontend/bower.rst (hunk 2 above)
Comment (cordoval) [inline cookbook/frontend/bower.rst]: committing assets totally defeats the purpose but is better than not saying anything. so :+1: @weaverryan, but would be nice a more clear message that you can also use npm alone and, despite of SO, successfully!

**Pull request merged**